frontend/src/components/common/header/header.tsx

```diff
@@ -0,0 +1,27 @@
+import { FC } from 'common/types/types';
+import { Image } from 'components/common/common';
+import logo from 'assets/img/logo.svg';
+import defaultUserAvatar from 'assets/img/avatar-default.png';
+
+import styles from './styles.module.scss';
+
+const Header: FC = () => (
+  <header>
+    <div className={styles['header-wrapper']}>
```

Comment:
Use camel case here

Reply:
hm, we have a lint rule to use snake case, or has something changed?

Reply:
Yes, we have changed that rule

Reply:
got it
merged development into branch and updated accordingly, thanks